src/06-time-tracking/time-tracking.apib

```diff
@@ -0,0 +1,268 @@
+## Time Tracking [/timeTracking]
+
+### timeTracking.list [GET /timeTracking.list]
+
+Get a list of tracked time.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `404c1777-6b61-4368-a388-0301dde8e72a`,`409e3446-96f2-4d9a-b20b-7f7cd055053e` (array[string], optional)
+            + user_id: `87982c96-f2fe-4b05-838c-ff42c0525758` (string, optional)
+            + work_type_id: `ce1287f0-0480-458d-b371-22862d864074` (string, optional)
+            + tracked_after: `2017-04-26T10:01:49+00:00` (string, optional)
```

Comment:
Q: We use `started_after` in the code. Should we update this?

Reply:
Yep, you're right. That's what the Jira ticket said and I didn't double check with the documentation.

I think `started_after` is a bit more explicit though.